src/diffusers/loaders/single_file.py

```diff
@@ -252,6 +252,7 @@ def from_single_file(cls, pretrained_model_link_or_path, **kwargs):
         local_files_only = kwargs.pop("local_files_only", False)
         revision = kwargs.pop("revision", None)
         torch_dtype = kwargs.pop("torch_dtype", None)
+        is_cosxl_edit = kwargs.pop("is_cosxl_edit", None)
```

Comment:
Not needed. 
https://github.com/huggingface/diffusers/blob/08bf7545070764d41637e90290ca9b92b392263e/src/diffusers/loaders/single_file.py#L317